use_cases/eluc/persistence/persistors/hf_persistor.py

```diff
@@ -0,0 +1,69 @@
+"""
+Persistor for models to and from HuggingFace repo.
+"""
+from pathlib import Path
+
+from huggingface_hub import HfApi, snapshot_download
+
+from persistence.serializers.serializer import Serializer
+
+class HuggingFacePersistor():
+    """
+    Persists models to and from HuggingFace repo.
+    """
+    def __init__(self, file_serializer: Serializer):
+        self.file_serializer = file_serializer
```

Comment:
`Persistor` wraps around a single `Serializer` and implements `persist` and `from_pretrained`